README.md

```diff
@@ -74,60 +66,58 @@ The following sections describe how to set up, build, and run the project.
 
 Make sure you have set up a development environment (that means, you’ve installed the CDS Compiler, Java, and Apache Maven) [as described here](https://cap.cloud.sap/docs/java/getting-started).
 
-## Clone Build & Run
+## Clone, Build & Run
 
 1.  Clone the project:
 
-```bash 
+  ```bash 
   git clone https://github.com/SAP-samples/cloud-cap-samples-java.git
-```
+  ```
 
 2. Build and run the application:
 
-```
+  ```
   mvn spring-boot:run
-```
+  ```
 
-## Using Eclipse
+## Using the Application    
 
-Optionally, use the following steps to import the project to Eclipse:
+Try it out using the following URLs:
+   
+-  <http://localhost:8080/>: shows the automatically generated index page of served paths.
+- <http://localhost:8080/fiori.html>: shows the graphical user interface
 
-1.  Import the project using **File > Import > Existing Maven Projects**.
-    
-    Now, you should see the projects **bookshop** and **bookshop-parent** in the project/package explorer view.
+You'll start with an empty stock of books as this procedure starts the bookshop application with an empty in-memory SQLite database.
 
-2.  In Project Explorer, change the property "Package Presentation" from "Flat" to "Hierarchical" for better understanding.
+Log in with one of the two mock users that are pre-defined for local development:
+  - To browse books, login in with user: `user`, password: `user`
+  - To manage bookd and orders, log in with User: `admin`, password: `admin`
 
```

Comment:
This could still fall under **Clone, Build & Run** so that we do not need the ## **Using the Application** Header